src/deadline/max_adaptor/MaxClient/render_handlers/default_max_handler.py

```diff
@@ -34,12 +34,29 @@ def __init__(self):
             "output_file_format": self.set_output_file_format,
             "state_set": self.set_state_set,
             "scene_file": self.set_scene_file,
+            # Render elements integration actions these setup the scene.
+            "enabled_modify_render_elements": self._no_op_action,
+            "configure_render_elements": self.configure_render_elements,
+            "cleanup_render_elements": self.cleanup_render_elements,
+            # Individual render element parameter actions.
+            # Currently the queue is populated but we do not take a specific action.
+            # This allows for customization for all derived renderers.
+            "render_elements_update_paths": self._no_op_action,
+            "render_elements_include_name_in_path": self._no_op_action,
+            "render_elements_include_type_in_path": self._no_op_action,
+            "render_elements_include_name_in_filename": self._no_op_action,
+            "render_elements_include_type_in_filename": self._no_op_action,
+            "vray_render_elements_vfb_control": self._no_op_action,
+            "vray_split_buffer_support": self._no_op_action,
+            "ignore_render_elements_by_name": self._no_op_action,
```

Comment:
Following from my previous comment on `_MAX_INIT_KEYS`, it may be cleaner to not send an action for now until we do need these